### PR TITLE
Improve maintenance mode Apache config

### PR DESCRIPTION
**Pull request by gido**
### 💬 Description
This changes allow us two things:

1. Use it into an `.htaccess` file
2. Use it outside the `.htaccess` file of the root directory (in a parent directory for example)

### CODE REVIEW

File: lib/capistrano/tasks/maintenance.rake [hunk 1]
```diff
@@ -83,14 +83,13 @@ namespace :maintenance do
...
-      info "<LocationMatch \"\\.(css|gif|ico|jpe?g|js|png|svg|webp|webm)$\">"
-      info "    ErrorDocument 503 \"Service unavailable.\""
-      info "</LocationMatch>"
+      info "ErrorDocument 503 \"Service unavailable.\""
       info "<IfModule mod_rewrite.c>"
       info "    <If \"! %{REQUEST_URI} =~ /\\.(css|gif|ico|jpe?g|js|png|svg|webp|webm)$/i && -f '%{DOCUMENT_ROOT}/#{fetch(:maintenance_basename)}'\">"
       info "        ErrorDocument 503 /#{fetch(:maintenance_basename)}"
       info "    </If>"
       info "    RewriteEngine On"
+      info "    RewriteOptions InheritDownBefore"
```
Comment (gido) [inline lib/capistrano/tasks/maintenance.rake:92]: the magic instruction that allow us to use this config into a parent `.htaccess`

**Comment by gido** [comment]
ping @yannlugrin @WengerK

**Review by WengerK — APPROVED** [review]

**Comment by WengerK** [comment]
So with this, I can add the maintence mode everywhere whitout having to interacte with the Hoster (infomaniak, ...) main htaccess ?

**Comment by gido** [comment]
> So with this, I can add the maintence mode everywhere whitout having to interacte with the Hoster (infomaniak, ...) main htaccess ?

Yes!
I did it on CLS hosting.

**Pull request merged**